app/models/landing_page/block/component.rb

```diff
@@ -0,0 +1,45 @@
+module LandingPage::Block
+  class Component < Base
+    ALLOWED_COMPONENTS = %w[
```

Comment:
I think I'd start very small with this list and expand it later if needed, rather than start big and then whittle down. There's a lot of components in this list that I'd have questions around how/if they'd be used on these pages.

For a starting list I'd suggest only action_link, big_number, cards, document_list, govspeak, heading, lead_paragraph, share_links - basically the ones we've implemented already.